### ISSUE TITLE
UPDATE: Resource page

### ISSUE THREAD

**Issue by vickimcfadden** [GSA/oasis-plus#443]

**Comment by rosamundtgov** [comment]
There are a couple of outstanding questions on this; @PowderD @ekidenda to confirm.

**Comment by rosamundtgov** [comment]
Outstanding questions have been answered - copy doc needs a couple of updates. Will post when finished or just make the update.

**Comment by klohman** [comment]
Copy doc is updated with Jon's responses - https://docs.google.com/document/d/1uaFZULg_IqJplogKvAXYwSpD9ST4hrBSG-21B8qnV_c/edit#heading=h.iygv7gu3pcbe

**Comment by rosamundtgov** [comment]
fact sheet

[24-00443_SS_24 PSC OASIS+ Fact Sheet_508C.pdf](https://github.com/GSA/oasis-plus/files/15504581/24-00443_SS_24.PSC.OASIS%2B.Fact.Sheet_508C.pdf)

**Comment by rosamundtgov** [comment]
put files in public/files

**Comment by klohman** [comment]
@rosamundtgov Is this ready for PSHC review?

**Comment by rosamundtgov** [comment]
@klohman yes. 

@ekidenda @PowderD take a look: https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/preview/gsa/oasis-plus/vickimcfadden-patch-1/

**Comment by ekidenda** [comment]
@klohman the shared link goes to the website landing page, not sure if that is the correct page to review, so could be an error...? I've reviewed the 'Resources' page instead and confirmed that the 'fact sheet' is linked and language has been updated according to the notes above. If these suffice, the task is complete and can be moved to done.

**Comment by klohman** [comment]
@ekidenda Yes this story relates to the [Resources page](https://federalist-76c8d2ee-2655-47f7-8fa4-63811842d0a5.sites.pages.cloud.gov/demo/gsa/oasis-plus/oasis-plus/resources/) (sorry for the confusion).

**Comment by ekidenda** [comment]
@klohman noted and in that case, moving this to done.
